includes/classes/Customer.php

```diff
@@ -645,6 +645,30 @@ public function getNumberOfOrders(): int
         return $result->fields['total'];
     }
 
+    /**
+     * Return a simple summary of orders for this customer, ordered most recent first.
+     *
+     * @param integer $limit Optional limit of the number of orders to return.
+     * @return array
+     */
+    public function getOrdersSummary(int $limit = 0): array {
+        global $db;
+        if (empty($this->customer_id)) {
+            return [];
+        }
+        $rs = $db->Execute("SELECT orders_id, date_purchased, orders_status_name
+            FROM " . TABLE_ORDERS . " o
+            INNER JOIN " . TABLE_ORDERS_STATUS . " os
+                ON o.orders_status = os.orders_status_id AND os.language_id = {$_SESSION['languages_id']}
+            WHERE customers_id = {$this->customer_id}
+            ORDER BY date_purchased DESC" . (!empty($limit) ? " LIMIT $limit" : '') . ';');
```

Comment:
The `$limit` can be passed as the second param to `$db->Execute($sql, $limit);` where it is also checked for validity.